ArduCopter/GCS_Mavlink.cpp

```diff
@@ -989,9 +989,11 @@ void GCS_MAVLINK_Copter::handleMessage(mavlink_message_t* msg)
                 if (!AP::ahrs().home_is_set()) {
                     break;
                 }
-                const Location &origin = copter.inertial_nav.get_origin();
+                Location origin;
                 pos_vector.z += AP::ahrs().get_home().alt;
-                pos_vector.z -= origin.alt;
+                if (copter.ahrs.get_origin(origin)) {
```

Comment:
Line just above this pokes AP::ahrs().  Should match it, I think.

Reply:
Could consider an early-return here, like we do if home isn't set - I'm not sure we should be doing this without an origin (and now we can test for it), which we couldn't before.